BestBuyCRUD/ISalesRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+/*
+namespace BestBuyCRUD
+{
+    interface ISalesRepository //declare classes
+    {
+        List<Sales> GetSales();
+        void CreateSales( Sales sales);
+        void DeleteSales(int saleId); //depends on the items chosen to delete 
+        void UpdateSales( Sales sales);
+
+    }
+}*/
```

Comment:
Commented-out code could be removed. If this interface is unused, its file could be deleted

Reply:
agree.  but those interefaces were created when Brandon was going over the topic of interface and would like to keep there for future references :)